projectile.el

```diff
@@ -4447,11 +4447,15 @@ project of that type"
       (projectile-read-command prompt default-cmd)
     default-cmd))
 
+(defcustom projectile-comint-mode nil
```

Comment:
Probably this should be named `projectile-use-compilation-shell-minor-mode` or something along those lines, as `comint-mode` is something else and that would be confusing IMO.

Reply:
Oh, well the description is actually a bit confusing and I don't have time to test this myself:

> If optional second arg COMINT is t the buffer will be in Comint mode with
‘compilation-shell-minor-mode’.

Reply:
It is both : when a prefix argument is used, the major mode of the compilation buffer is `comint-mode`, and the minor mode `compilation-shell-minor-mode` is activated. `comint-mode` is the more important one, though. All `compilation-shell-minor-mode` does is rebind keys to avoid conflicts : 

```
When Compilation Shell minor mode is enabled, all the
error-parsing commands of the Compilation major mode are
available but bound to keys that don't collide with Shell mode.
```

Perhaps `projectile-comint` would be the best choice of name : it uses `comint`, which is the name of the argument of the `compile` function and of the major mode the compilation buffer will be in, and avoids the `-mode` prefix, which might wrongly imply that this is an actual mode. Would you be fine with that ?

Reply:
For completeness's sake, here are the bindings defined in `compilation-shell-minor-mode-map` :

```
C-M-n  compilation-next-error
C-M-p  compilation-previous-error
M-RET  compile-goto-error
M-{    compilation-previous-file
M-}    compilation-next-file
```

Reply:
Got it. Okay, let's name this `projectile-compilation-use-comint-mode` or something like this. I just want to make it a bit more explicit that this is related to complition commands.